app/src/androidTest/java/org/mozilla/fenix/ui/robots/NavigationToolbarRobot.kt

```diff
@@ -28,7 +29,6 @@ class NavigationToolbarRobot {
         val mDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
 
         fun enterURLAndEnterToBrowser(url: Uri, interact: BrowserRobot.() -> Unit): BrowserRobot.Transition {
-            mDevice.wait(Until.findObject(By.text("Search or enter address")), waitingTime)
```

Comment:
for example without this wait the test might fail looking for urlBar?

Reply:
For this one in particular we can't use "Search or enter address." as that text is only initially visible and this function is reused.